src/ray/core_worker/test/direct_task_transport_mock_test.cc

```diff
@@ -46,7 +46,8 @@ class DirectTaskTransportTest : public ::testing::Test {
         WorkerType::WORKER, /* worker_type */
         0,                  /* lease_timeout_ms */
         actor_creator,
-        JobID::Nil() /* job_id */);
+        JobID::Nil() /* job_id */,
+        []() { return 1; });
```

Comment:
This is hard to read.. consider making a `kDefaultXyz` or at least add a comment.

Reply:
Same thing for `direct_task_transport_test.cc`. Since the value returned varies, can have a simple `createMaxPendingLeaseRequestsPerSchedulingCategoryCallback` (naming could use definitely use simplification 😄 ).